src/main/java/com/code_intelligence/jazzer/autofuzz/BUILD.bazel

```diff
@@ -11,6 +11,7 @@ java_library(
     visibility = ["//visibility:public"],
     deps = [
         "//src/main/java/com/code_intelligence/jazzer/api",
+        "//src/main/java/com/code_intelligence/jazzer/runtime:jazzer_bootstrap_compile_only",
```

Comment:
@fmeum `HardToCatchError` is injected into the bootstrap classloader, so this should be the correct dependency, right?

Reply:
Yeah, that's the correct one.